LibNFS.xs

```diff
@@ -983,6 +983,33 @@ static void _set_unix_authn(pTHX_ struct nfs_context* nfs, SV* value_sv) {
     nfs_set_auth(nfs, auth);
 }
 
+#ifdef NLNFS_NFS_SET_READDIR_MAX_BUFFER_SIZE
+static void _set_readdir_max_buffer_size(pTHX_ struct nfs_context* nfs, SV* value_sv) {
+
+    if (SvROK(value_sv) && (SvTYPE(SvRV(value_sv)) == SVt_PVAV)) {
+        AV* av = (AV*) SvRV(value_sv);
+        uint32_t nums_count = 1 + av_len(av);
+        if (nums_count != 2) croak("“%s” must contain exactly 2 numbers", "readdir_buffer");
+
+        uint32_t nums[nums_count];
```

Comment:
Might as well hard-code this at 2 in order to avoid a variadic array. Maybe create a #define for it, e.g.:
```
#define MAX_BUFFER_SIZE_ARRAY_SIZE 2
```
… then:
```
    UV nums[MAX_BUFFER_SIZE_ARRAY_SIZE];
```

Reply:
I had it hard-coded first - then thought if there's any need to allow a different number of arguments for some reason (or the code gets copied), it should react dynamically. I can change it though if you like.

Reply:
AFAICT, as the code stands there would still need to be changes if there were a case for something besides 2 arguments.

Reply:
Done